client/dashboard/profile-wizard/steps/business-details.js

```diff
@@ -139,18 +139,18 @@ class BusinessDetails extends Component {
 		return keys( pickBy( values ) ).filter( name => this.extensions.includes( name ) );
 	}
 
-	getNumberRangeString( min, max = false, format = numberFormat ) {
+	getNumberRangeString( min, max = false, format = formatValue ) {
 		if ( ! max ) {
 			return sprintf(
 				_x( '%s+', 'store product count or revenue', 'woocommerce-admin' ),
-				format( min )
+				format( 'number', min )
```

Comment:
If you didn't want to change the method signature here (since that's an API change for this function) you could use `partial` to compose it with the first argument when setting the parameter default. Though.. I don't see any calls to `getNumberRangeString()` that override `format` - perhaps its unnecessary?

Reply:
Good point.  It doesn't appear to be necessary so I've removed for now (maybe something was removed where it was previously neeeded), but /cc @justinshreve just to double check where this was being used.